.github/workflows/int-and-connected-test-run.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       - name: "Checkout code"
         uses: actions/checkout@v2
       - name: "Run Connected test via Gradle"
-        uses: broadinstitute/datarepo-actions/actions/main@0.39.0
+        uses: broadinstitute/datarepo-actions/actions/main@rc-se-actions-testing
```

Comment:
should this be using what looks like a personal branch?

Reply:
Doh, Muscles already caught this, ignore me.

Reply:
(my guess is @samanehsan / @RoriCremer will switch back when ready)